src/main/java/uk/gov/hmcts/darts/Application.java

```diff
@@ -45,9 +48,7 @@ public static void main(final String[] args) {
     public void run(String... args) throws Exception {
         if (System.getenv("ATS_MODE") != null) {
             log.info("ATS_MODE activated");
-            // Temporary workaround to prevent many media requests in OPEN state
-            var openAudioRequests = mediaRequestService.getMediaRequestsByStatus(AudioRequestStatus.OPEN);
-            mediaRequestService.updateAudioRequestStatus(openAudioRequests.get(0).getId(), AudioRequestStatus.PROCESSING);
+            audioTransformationService.handleKedaInvocationForMediaRequests();
```

Comment:
Should we expose this method out via an interface in the `audio` package, instead of using the service directly? CC @chrisbellingham-hmcts

Reply:
We already have AudioTransformationService Interface which the logic should be using. Since the requirement is to have an asynchronous process which will be triggered by KEDA, so I don't see a reason to expose it as Api. Unless there is a requirement to trigger the process forcefully from portal.

Reply:
I didn't mean exposing it as an API, just an interface into the service. As was proposed very early on in the development of the app. In order to make clear the cross-package calls that are happening.

Reply:
My interpretation of @davet1985's comment is that we should follow the established pattern whereby any external usages of a service method should be defined in an interface in the `audio.api` package. This will not be an http api, simply a java interface with an implementation that delegates to the service.

I think this is a good point, and I agree we should follow that pattern here.

Reply:
I see. I misinterpreted as exposing it as an http api. Yes we can follow the same pattern then.